qubes_config/global_config.glade

```diff
@@ -7283,7 +7188,7 @@ For more information, &lt;a href="https://www.qubes-os.org/doc/certified-hardwar
                               <object class="GtkLabel" id="thisdevice_security_info">
                                 <property name="visible">True</property>
                                 <property name="can-focus">True</property>
-                                <property name="label" translatable="yes">To learn more about Qubes OS Security Report, you can check the &lt;a href="https://github.com/Qubes-Community/Contents/blob/master/docs/common-tasks/opening-urls-in-vms.md"&gt;online documentation&lt;/a&gt;. 
```

Comment:
This link doesn't seem to match the context. Maybe a copy/paste error?

Reply:
Yup, I think it's a result of "hey we should add documentation on it" and me forgetting to actually add it. I made a task for it: https://github.com/QubesOS/qubes-issues/issues/8165